src/ncp/spinel.h

```diff
@@ -346,6 +348,58 @@ typedef enum
     SPINEL_PROP_HBO_MEM_MAX             = 10,       ///< Max offload mem [S]
     SPINEL_PROP_HBO_BLOCK_MAX           = 11,       ///< Max offload block [S]
 
+    SPINEL_PROP_BASE_EXT__BEGIN         = 0x1000,
+
+    /// Available GPIO Bitmask
+    /** Format: `D`
+     *  Type: Read-Only
+     *
+     * Contains a bit field identifying which GPIOs are supported. Cleared bits
+     * are not supported. Set bits are supported.
+     */
+    SPINEL_PROP_GPIO_AVAILABLE          = SPINEL_PROP_BASE__BEGIN + 0,
```

Comment:
Looks like `SPINEL_PROP_BASE__BEGIN` should be `SPINEL_PROP_BASE_EXT__BEGIN`, right?